client/packages/system/src/Encounter/DetailView/DetailView.tsx

```diff
@@ -75,9 +76,20 @@ export const DetailView: FC = () => {
   useEffect(() => {
     if (encounter)
       setSuffix(
-        `${
-          encounter.document.documentRegistry?.name
-        } - ${dateFormat.localisedDateTime(encounter.startDatetime)}`
+        <>
+          <Breadcrumb
+            to={RouteBuilder.create(AppRoute.Dispensary)
+              .addPart(AppRoute.Patients)
+              .addPart(encounter.patient.id)
+              .build()}
+            key={'patient'}
+          >
+            {`${encounter.patient.firstName} ${encounter.patient.lastName}`}
```

Comment:
Something we might need to consider -- some languages require the family name to go first. We might need to add a "showFullName" utility function to our localisation module to return an appropriately formatted full name for the language.

I'll make an issue for further discussion.

Reply:
#1565 :)

Reply:
Should also create a utility function for the patient name. And as part of that, handle nulls e.g. 

<img width="578" alt="image" src="https://user-images.githubusercontent.com/9192912/232977946-6d62a10c-cc2c-4c8f-9b7f-cf9483a43dcd.png">

Reply:
yeah, that is nicer. the functions aren't all hooks in the IntlUtils tho - so I would just expose the hook based items. Otherwise you are forcing the consumer to use the utils as a hook, and that might be annoying? I guess check current usage and see.

Reply:
#1583 😎